<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGUI">
      <attribute name="title">
       <string>GUI</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="labelStyle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Style</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QComboBox" name="comboBoxStyle"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelAppFont">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Font</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QFontComboBox" name="fontComboBoxAppFont"/>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="labelAppFontSize">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QSpinBox" name="spinBoxAppFontSize"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelLogFont">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Log font</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QFontComboBox" name="fontComboBoxLogFont"/>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="labelLogFontSize">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QSpinBox" name="spinBoxLogFontSize"/>
       </item>
       <item row="3" column="0" colspan="4">
        <spacer name="verticalSpacerGUI">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>331</width>
           <height>3</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPaths">
      <attribute name="title">
       <string>Paths</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="labelChdmanBinary">
         <property name="text">
          <string>CHDMAN binary</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditChdmanBinary">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="toolButtonBrowseChdmanBinary">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="icon">
          <iconset resource="qchdman.qrc">
           <normaloff>:/data/img/fileopen.png</normaloff>:/data/img/fileopen.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <spacer name="verticalSpacerPaths">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>comboBoxStyle</tabstop>
  <tabstop>fontComboBoxAppFont</tabstop>
  <tabstop>spinBoxAppFontSize</tabstop>
  <tabstop>fontComboBoxLogFont</tabstop>
  <tabstop>spinBoxLogFontSize</tabstop>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>lineEditChdmanBinary</tabstop>
  <tabstop>toolButtonBrowseChdmanBinary</tabstop>
 </tabstops>
 <resources>
  <include location="qchdman.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
